Doc/whatsnew/3.11.rst

```diff
@@ -262,15 +272,20 @@ See :pep:`655` for more details.
 (Contributed by David Foster and Jelle Zijlstra in :issue:`47087`. PEP
 written by David Foster.)
 
+
+.. _whatsnew311-pep673:
+
 PEP 673: ``Self`` type
 ----------------------
 
 The new :data:`~typing.Self` annotation provides a simple and intuitive
 way to annotate methods that return an instance of their class. This
-behaves the same as the :data:`~typing.TypeVar`-based approach specified
-in :pep:`484` but is more concise and easier to follow.
+behaves the same as the :data:`~typing.TypeVar`-based approach
+:pep:`specified in PEP 484<484#annotating-instance-and-class-methods>`,
```

Comment:
```suggestion
:pep:`specified in PEP 484 <484#annotating-instance-and-class-methods>`,
```

Reply:
Oops, don't know how I did that...rather embarrassing. Thanks!